guides/common/modules/proc_adding-kubevirt-connection.adoc

```diff
@@ -12,28 +12,30 @@ Use this procedure to add {KubeVirt} as a compute resource in {Project}.
 # {foreman-installer} --enable-foreman-plugin-kubevirt
 ----
 
-. Generate a bearer token to use for HTTP and HTTPs authentication.
-On the {KubeVirt} server, list the secrets that contain tokens:
+. Obtain a token to use for HTTP and HTTPs authentication:
+
+.. Log in to the {Kubernetes} cluster and list the secrets that contain tokens:
 +
+[options="nowrap" subs="+quotes,attributes"]
 ----
-# kubectl get secrets
+$ {kubevirt-command} get secrets
 ----
 
-. List the token for your secret:
+.. Obtain the token for your secret:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# kubectl get secrets _YOUR_SECRET_ -o jsonpath='{.data.token}' | base64 -d | xargs
+$ {kubevirt-command} get secrets _YOUR_SECRET_ -o jsonpath='{.data.token}' | base64 -d | xargs
```

Comment:
```suggestion
$ {kubevirt-command} get secrets _MY_SECRET_ -o jsonpath='{.data.token}' | base64 -d | xargs
```
I know this has been here before, but generally we use "MY" when substituting user input.